app/assets/javascripts/directives/configuration/all_time_check.js

```diff
@@ -29,19 +29,19 @@ ManageIQ.angular.app.directive('allTimeCheck', function() {
           if (hoursArr[j] === name) {
             updatedHourArr = allQuarterArrHoursChecked(name, i, value);
           } else {
-            if (!angular.equals(_.times(6, _.constant(true)), scope[scope.model][hoursArr[j]])) {
+            if (!angular.equals(_.times(6, _.constant(true)), scope.vm[scope.vm.model][hoursArr[j]])) {
```

Comment:
Seems like this directive mostly uses `scope.vm.timeProfileModel` .. so maybe you can change all these `scope.vm[scope.vm.model]` to just be `scope.vm.timeProfileModel`? :)